packages/commons-server/src/libs/utils.ts

```diff
@@ -108,6 +108,13 @@ export function CreateTransaction(
   response: Response
 ): Transaction {
   const requestUrl = new URL(request.originalUrl, 'http://localhost/');
+    let queryString = requestUrl.search.slice(1);
+    try {
+        queryString = decodeURI(queryString);
+    } catch (err) {
+        queryString = '';
+        console.error('ERROR DECODING URI: ', err instanceof Error ? err.message : err);
```

Comment:
One remark: the `console.error` will not log errors using the correct transport (winston for the CLI, and electron-log for the desktop app) and I think info would be lost at least in the desktop app once it's packaged. We could pass the logger instance to the `CreateTransaction` function but it is not always available with the current architecture, especially in the desktop app where the server emit errors rather than logging directly. 
Also, I am not sure logging here bring much value. As a counterexample, when there is a parse error with headers we directly put the error message in the header value, so the user can immediately see the problem in the most obvious place.
Another thing is that when there is a Malformed URI error, the raw query params will be empty, while the structured query params (from Express' `request.query`) will be shown:
![image](https://user-images.githubusercontent.com/7489814/169489843-c7bf7775-60a4-4fe6-80e6-7d34a6c9e933.png)

So, I suggest that we revert to the raw query string in the `catch` maybe prefixed by something like "[Error: URI malformed]":
![image](https://user-images.githubusercontent.com/7489814/169491618-b73b8d70-59a3-49f2-8282-7779ac3ac803.png)
I think there is no reason we should not be able to see the raw version even if malformed. What do you think? 

```suggestion
        queryString = `[Error URI malformed]: ${queryString}`;
```